integration/shareddata/shareddata.go

```diff
@@ -55,7 +55,6 @@ func AllProviders() Providers {
 		ObjectIDs,
 		Bools,
 		DateTimes,
-		Nulls,
```

Comment:
> // AllProviders returns all providers in random order.

It still should do that. And I think incompatible provider should be skipped automatically by setup.

Reply:
Ok, sounds reasonable!

Reply:
fixed